### PR TITLE
Refactor helm template

### PR DESCRIPTION
**Pull request by gkuga**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/pipecd/values.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Service and ingress.
 service:
-  port: 443
-  internalPort: 8443
+  port: 80
```
Comment (nghialv) [inline manifests/pipecd/values.yaml:3]: `8080`

File: manifests/pipecd/templates/deployment.yaml [hunk 2]
```diff
@@ -90,7 +90,7 @@ spec:
           imagePullPolicy: IfNotPresent
           args:
           - server
-{{- if or .Values.quickstart .Values.api.args.insecureCookie }}
+{{- if or .Values.quickstart .Values.api.args.insecureCookie (eq (int .Values.service.port) 80 ) }}
```
Comment (nghialv) [inline manifests/pipecd/templates/deployment.yaml:93]:
```
{{- if not .Values.api.args.secureCookie }}
- --insecure-cookie=true
{{- end }}
```

Comment (nghialv) [reply manifests/pipecd/templates/deployment.yaml:93]: And please update the doc at 
https://github.com/pipe-cd/pipe/blob/master/docs/content/en/docs/operator-manual/control-plane/installation.md

Comment (gkuga) [reply manifests/pipecd/templates/deployment.yaml]: I addressed it. PTAL :pray:

**Comment by nghialv** [comment]
Thank you.
/approve

**Pull request merged**